package.json

```diff
@@ -56,12 +57,8 @@
     "@uniswap/v2-core": "^1.0.1",
     "@uniswap/v2-periphery": "^1.1.0-beta.0",
     "canonical-weth": "^1.4.0",
-    "cross-fetch": "^4.0.0",
     "eth-permit": "^0.2.3",
     "ethereum-sources-downloader": "^0.1.21",
-    "node-fetch": "^3.3.2",
-    "scribble-generator": "^1.1.0",
```

Comment:
These are not explicitly needed, scribble needs to be installed as a global env anyway if anybody wants to run it.